content/support.md

```diff
@@ -6,7 +6,11 @@ description: Learn about support options available in Galaxy
 
 All Galaxy customers can submit support cases from within Galaxy and receive Basic online support for deployment-related questions. There is no guaranteed response SLA for Basic support but questions are typically answered within 1-2 business days.  
```

Comment:
We now offer a 1 business support SLA for all Galaxy accounts for non-critical tickets.